src/Installer/Beta.htm

```diff
@@ -161,5 +162,6 @@
 				<li>FLEx Bridge: Initial release.</li>
 			</ol></li>
 		</ul>
```

Comment:
The whole file including the div contents will end up being replaced the next time we run the release artifact task on it, so I didn't feel like indenting all the contents. I just added the div so that the task would work properly when that time comes.